### DISCUSSION TITLE
Thoughts about `light-dark()` css function

### DISCUSSION THREAD

**Discussion by itsmnthn** [unocss/unocss#3704] [General]
This seems a gud way to manage [light dark ](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/light-dark) modes with zero runtime JS to switch modes. However the support for this function is not adapted by all browsers. But something to look forward in the future.

**Comment by desvandev** [comment]
Love that new function, but reading through chromium looks like it has some issues still. I would let this marinate a bit more.
